Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -1541,44 +1541,38 @@ DEFINE_TO_FROM_CSS_VALUE_ID_FUNCTIONS
 #undef TYPE
 #undef FOR_EACH
 
-constexpr CSSValueID toCSSValueID(TextWrap wrap)
+constexpr CSSValueID toCSSValueID(TextWrapMode wrap)
```

Comment:
Why can’t we `DEFINE_TO_FROM_CSS_VALUE_ID_FUNCTIONS` for this? Surprised it was hand-written.

Reply:
it's because the enum value is NoWrap and the CSSValueID is CSSValueNowrap (notice the capitalization).